src/cmd-diff

```diff
@@ -78,7 +78,7 @@ def main():
         args.diff_from = builds.get_previous()
         args.diff_to = latest_build
     elif args.diff_from is None:
-        args.diff_from = latest_build
+        args.diff_from = builds.get_previous()
```

Comment:
Hmm, what were you trying to do that surprised you and motivated this?

E.g. if I do `cosa diff --to=$build` I expect the from to be the latest build. This matches `git diff` semantics. Having it actually diff against the previous build is confusing.

Reply:
See the last commit in https://github.com/coreos/coreos-assembler/pull/4327

Basically I wanted to be able to not have to specify the previous build in `cmd-build-with-buildah` and have `cosa diff` detect that for me.

It also enables things like being able to run `cosa diff --rpms` without having to specify a `--to` and a `--from` at all, like `rpm-ostree db diff` does if you don't provide commits to it.

Maybe the semantic should be "if `to` is `latest_build` then `from` will default to previous build, otherwise default to `latest_build`".

Reply:
> It also enables things like being able to run cosa diff --rpms without having to specify a --to and a --from at all, like rpm-ostree db diff does if you don't provide commits to it.

Hmm, that's already the case today, no?

ISTM like in the last commit of #4327, we can just not pass in `--to` or `--from` at all in the default case. If there's a parent build, just passing `--from` should do the right thing.